install_test_deps.sh

```diff
@@ -0,0 +1,6 @@
+# Installs xblock-sdk and dependencies needed to run the tests suite.
+# Run this script inside a fresh virtual environment.
+pip install -e git://github.com/edx/xblock-sdk.git#egg=xblock-sdk
+pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements.txt
+pip install -r $VIRTUAL_ENV/src/xblock-sdk/test-requirements.txt
+python setup.py develop
```

Comment:
It wouldn't - `pip install` does not recursively install `requrement.txt` files and while installing xblock-sdk with pip does install the dependencies specified in xblock-sdk's `setup.py` file, that file does not list the git dependencies from `requirements.txt`.